ddtrace/ext/__init__.py

```diff
@@ -19,7 +19,7 @@ class SpanTypes(Enum):
     WORKER = "worker"
 
 
-@removals.removed_class("AppTypes")
+@removals.removed_class("AppTypes", version="1.0.0")
```

Comment:
🤔 I think we can just remove this since we're removing the constants below

Reply:
oh yeah, probably not necessary, but also doesn't hurt ?

Reply:
do you mean just the `removals.removed_class()` decorator?

Reply:
Removing the whole class - or at least that was my take away from the previous discussion (since it's been 16 months since it's been deprecated and the instructions were clear, etc)

Reply:
Right, in that case I agree I think it's fair to get rid of the class entirely.